docs/miscellaneous.rst

```diff
@@ -168,14 +169,14 @@ These steps are applied to each basic block and the newly generated code is used
     else
       return 1;
 
-is simplified to code which can also be compiled from
+Is simplified to code which can also be compiled from:
```

Comment:
If it starts with a capital letter it should be a full sentence, but it is not.

Reply:
@chriseth I disagree here, and I was unsure of any 'official rule' on this I asked around and got a general opinion that for a code block that is longer than a line or two, you should start a new sentence.

So… As a compromise 🙂, I clarified the text leading in and out a little.